tools/twix/src/panels/map/layers/behavior_simulator.rs

```diff
@@ -12,23 +12,33 @@ use crate::{
 
 pub struct BehaviorSimulator {
     robot_to_field: PlayersValueBuffer,
+    motion_command: PlayersValueBuffer,
 }
 
 impl Layer for BehaviorSimulator {
     const NAME: &'static str = "Behavior Simulator";
 
     fn new(nao: Arc<Nao>) -> Self {
         let robot_to_field = PlayersValueBuffer::try_new(
-            nao,
+            &nao,
```

Comment:
Arcs are cheap to clone. Please use them directly instead of a reference.
```suggestion
            nao,clone(),
```

Reply:
done